scripts/provisioning/s3setup_prereqs.json

```diff
@@ -52,8 +52,9 @@
     "rpms": ["openldap-servers", "openldap-clients", "haproxy", "cortx-motr", "python36-ldap"],
     "pip3s": ["s3confstore", "cryptography", "s3cipher"]
   },
-  "test": {
-    "rpms": ["openldap-servers", "openldap-clients", "haproxy", "cortx-motr", "python36-ldap"],
+  "test" :
+  {
+    "rpms": ["openldap-servers", "openldap-clients", "haproxy", "cortx-motr", "python36-ldap", "cortx-s3-test"],
```

Comment:
cortx-s3-test will not be part of Prod. So don't you think post_install will fail for production builds !!

Reply:
In fact test can also be run on a different node as well (outside cluster).
Better remove all rpms and pip3s except s3-test RPM

Reply:
Removed haproxy and cortx-motr